package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "yii2",
+  "version": "2.0.10",
```

Comment:
is this "version" key needed? I'd remove it to avoid the need of maintaining it.